examples/python/src/utils/frontify_queries/create_asset/create_asset.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+from src.services.FrontifyClient import FrontifyClient
+from src.utils.frontify_queries.create_asset.create_asset_executor import (
+    create_asset_executor,
+)
+
+
+def create_asset(
+    client: FrontifyClient,
+    projectid: str,
+    fileid: str,
+    title: str = None,
+    filename: str = None,
+    directory: str = None, # directory path in the format "folder1/folder2"
+    description: str = "",
+    author: str = "",
+    copyright: str = None,
+    copyright_notice: str = None,
+    tags: list = None,
+    expires_at: str = None,
+    externalId: str = "",
+    skip_exif: bool = True,
+) -> str:
+    """
+
+    :rtype: object
+    """
+    if filename is None and title is None:
+        raise ValueError("Either 'filename' or 'title' must be provided.")
+    
+    create_asset_input = {
+        "input": {
+            "projectId": projectid,
```

Comment:
```suggestion
            "parentId": parentid,
```
projectid is deprecated